examples/runtime/starlark/inline/README.md

```diff
@@ -0,0 +1,43 @@
+# Starlark Inline Example
+
+In this example, we are going to demonstrate how to execute a starlark script
+in starlark function against the input KRM resources to update or validate them.
```

Comment:
This paragraph is too generic/ambiguous. What exactly is this example doing?

Keep in mind we don't need this to be a general introduction to Starkark, so lead by example.

Reply:
https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/183

Reply:
Updated the example to follow the template.
Link to reference doc part has been skipped since we don't have it live.